src/content/styles/Button.scss

```diff
@@ -170,24 +170,23 @@ $square-button-height-width: 40px;
   width: $square-button-height-width;
   height: $square-button-height-width;
 
-  margin: 0;
-  margin-right: 10px;
+  margin: 0  1vw 0 0;
   left: 0;
   right: 0;
+  top: 0;
 
-  align-items: center;
-  text-align: center;
-  text-transform: capitalize;
+  //align-items: center;
+  //text-align: center;
+  //text-transform: capitalize;
 
   color: $bright_blue;
   background-color: transparent;
   border: 3px solid $bright_blue;
   border-radius: 10px;
 
-  display: flex;
-  position: relative;
-  justify-content: center;
-  top: 0px;
+  //display: flex;
+  //position: relative;
+  //justify-content: center;
```

Comment:
Noticed these are commented out, should we delete them?

Reply:
Yeah they were redundant